### PR TITLE
Update cohost

### PR DESCRIPTION
**Pull request by jkap**
SOURCE: I merged it this morning.

### CODE REVIEW

File: entries/c/cohost.org.json [hunk 1]
```diff
@@ -1,12 +1,7 @@
...
     "domain": "cohost.org",
-    "contact": {
-      "email": "support@cohost.org",
-      "twitter": "cohost_org"
-    },
-    "categories": [
-      "social"
-    ]
+    "tfa": ["totp"],
+    "categories": ["social"]
```
Comment (Carlgo11) [inline entries/c/cohost.org.json:5]: Please include one array value per line as this is the code style used for all the other files in the repo.

**Review by Carlgo11 — CHANGES_REQUESTED** [review]
I see that you haven't add any documentation link. Could you maybe find one that describes how to enable 2FA on the site?

If no documentation is available or you can't find it, please attach some screenshots of the 2FA setup process and/or login process.
> **Note** Remember to blur or crop out any personal information linked to you.

You can attach the screenshots here in a comment, like this:
screenshot

**Comment by hkamran80** [comment]
cohost - 2FA

**Comment by Carlgo11** [comment]
As the requested changes haven't been resolved yet I'm going to label this PR as inactive.
> **Warning**
> Unless there's activity within the next 7 days I'll close the PR.

**Comment by Carlgo11** [comment]
/
Closing due to inactivity.
> **Note**
> The PR author you can still commit the requested changes and re-open this PR.
> If someone else wants to commit the requested changes you can create a new PR and link to this PR.

**Pull request closed without merging**